Blog - best mobile app analytics

Lior539 opened the pull request:
## Article checklist

- [x] I've added (at least) 3-5 internal links to this new article
- [x] I've added keywords for this page to the rank tracker in Ahrefs
- [x] I've checked the preview build of the article
- [x] The date on the article is today's date

andyvan-ph reviewed (changes requested):
I know this is based on the [mobile A/B testing](https://posthog.com/blog/best-mobile-app-ab-testing-tools) template we already have, but this feels a bit light atm. We can do better.

Thoughts:
- Outside of the short 'What is?' sections, we're not really communicating the features of each tool super well – i.e. what's the meaningful difference between us, Mixpanel and UXCam, for example?

- We include platforms that they support, but not what features are available on each SDK / platform. Probably worth clarifying this for session replay specifically.

- Maybe we should ditch the bullet points lists (both) for a more detailed feature table to cover the above?

-  I think we can lose the bits at the end – i.e. Is posthog right for you, further reading and the CTA.

Not sure why, but preview is rendering incorrectly too – no headline, or blog template. Probably a bug somewhere in the PR.

![Screenshot 2024-07-15 at 14 37 35](https://github.com/user-attachments/assets/1a55d7e2-8dda-4429-bc03-fe46f0fdce82)

jtemperton reviewed (commented):
Will take a look once you've worked on @andyvan-ph's feedback to avoid confusing things.

Lior539 commented:
Reworked the list:
- Dropped "which platforms do they support"
- Added "Why do companies choose","Key features" and "bottom line" like we have in the alternatives guide
- Dropped the bulleted list and just kept "best for"

jtemperton commented on contents/blog/best-mobile-app-analytics-tools.mdx:
```diff
@@ -0,0 +1,327 @@
+import { ComparisonTable } from 'components/ComparisonTable'
+import { ComparisonRow } from 'components/ComparisonTable/row'
+
+When it comes to mobile apps, there are many different analytics tools to choose from, but each one is built with a specific target audience in mind. In this guide, we’ll explore these options so that you can decide which is best for you.
```
```suggestion
When it comes to mobile apps, there are a bunch of different analytics tools to choose from, but each is built with a specific target audience in mind. In this guide, we’ll explore the options so that you can decide which is best for you.
```

jtemperton commented on contents/blog/best-mobile-app-analytics-tools.mdx:
```diff
@@ -0,0 +1,327 @@
+
+### How much does PostHog cost?
+
+PostHog has [transparent pricing](/pricing) based on the usage. It’s free to get started and completely free for the first 1 million analytics events. After this free monthly allowance, pricing starts at $0.000248/event (or $2.48 for 10k events), and events cost progressively less the more you use. You can also set billing limits to ensure you don't get surprise bills. 
```
```suggestion
PostHog has [transparent pricing](/pricing) based on usage. It’s free to get started and completely free for the first 1 million analytics events. After this free monthly allowance, pricing starts at $0.000248/event (or $2.48 for 10k events), and events cost progressively less the more you use. You can also set billing limits to ensure you don't get surprise bills. 
```

jtemperton commented on contents/blog/best-mobile-app-analytics-tools.mdx:
```diff
@@ -0,0 +1,327 @@
+
+> #### Bottom line
+> 
+> For teams looking for all the tools they need to improve their products, PostHog makes for a great alternative. This is especially true for startups and scaleups thanks to it having a generous free tier.
```
a great alternative to what? think this might be a leftover line from another guide?

Lior539 replied:
```suggestion
> For teams looking for all the tools they need to improve their products, PostHog makes for a great choice. This is especially true for startups and scaleups thanks to it having a generous free tier.
```

jtemperton commented on contents/blog/best-mobile-app-analytics-tools.mdx:
```diff
@@ -0,0 +1,327 @@
+
+### Why do companies use Mixpanel?
+ 
+Looking at [G2 reviews](https://www.g2.com/products/mixpanel/reviews), companies choose Mixpanel to:
```
```suggestion
Looking at [G2 reviews](https://www.g2.com/products/mixpanel/reviews), companies choose Mixpanel because:
```

Lior539 replied:
Doesn't `to` make more sense here because of the sentences that follow below?

jtemperton commented on contents/blog/best-mobile-app-analytics-tools.mdx:
```diff
@@ -0,0 +1,327 @@
+
+- 📺 **Session replays:** View exactly how users are using your app. Includes event timelines, console logs, network activity, and 90-day data retention.
+
+- 🔥 **Heatmaps:** See where users tap, what point they scroll to, and the areas that get the most attention.
```
```suggestion
- 🔥 **Heatmaps:** See where users tap, where they scroll to, and the areas that get the most attention.
```

jtemperton commented on contents/blog/best-mobile-app-analytics-tools.mdx:
```diff
@@ -0,0 +1,327 @@
+
+### Why do companies use UXCam?
+ 
+Looking at [G2 reviews](https://www.g2.com/products/uxcam/reviews), companies choose UXCam to:
```
```suggestion
Looking at [G2 reviews](https://www.g2.com/products/uxcam/reviews), companies choose UXCam because:
```

Lior539 replied:
Same as above re using `to`?

jtemperton commented on contents/blog/best-mobile-app-analytics-tools.mdx:
```diff
@@ -0,0 +1,327 @@
+ 
+Looking at [G2 reviews](https://www.g2.com/products/uxcam/reviews), companies choose UXCam to:
+
+1. **Insights on user behavior:** Shows what they are doing, where their attention is going, and how they are spending their time. Users love the realizations they get with UXCam's tools.
```
```suggestion
1. **They get insights on user behavior:** Insights show what users are doing, where their attention is going, and how they are spending their time. Users love the realizations they get with UXCam's tools.
```

jtemperton commented on contents/blog/best-mobile-app-analytics-tools.mdx:
```diff
@@ -0,0 +1,327 @@
+
+2. **It's easy to use:** Reviewers note how easy UXCam is to set up and use. This means they can go from idea to actionable learnings quickly.
+
+3. **Heatmaps and scrollmaps:** Seeing clicks and scroll depth down works well and creates actionable insights for design and product teams. Helps improve UX and page designs.
```
```suggestion
3. **Heatmaps and scrollmaps:** Seeing clicks and scroll depth works well and creates actionable insights for design and product teams. Helps improve UX and page designs.
```

jtemperton commented on contents/blog/best-mobile-app-analytics-tools.mdx:
```diff
@@ -0,0 +1,327 @@
+
+### How much does UXCam cost?
+
+UXCam's pricing is sales-driven and completely opaque, so it's not clear how much it costs. Their sales team will give you a quote after you've spoken to them.
```
```suggestion
UXCam's pricing is sales-driven and completely opaque, so it's not clear how much it costs. The company's sales team will give you a quote after you've spoken to them.
```

jtemperton commented on contents/blog/best-mobile-app-analytics-tools.mdx:
```diff
@@ -0,0 +1,327 @@
+
+> #### Bottom line
+>
+> Like PostHog, Amplitude is a good alternative if you value powerful analytics and experimentation in one. If you don't need session replay or heatmaps, it can be a good choice.
```
alternative to _what_?

should we be saying it's a good option/choice/something

Lior539 replied:
```suggestion
> Like PostHog, Amplitude is a good option if you value powerful analytics and experimentation in one. If you don't need session replay or heatmaps, it can be a good choice.
```

jtemperton commented on contents/blog/best-mobile-app-analytics-tools.mdx:
```diff
@@ -0,0 +1,327 @@
+
+According to [G2 reviews](https://www.g2.com/products/firebase/reviews), companies choose Firebase because:
+
+1. **It's a comprehensive toolkit:** Firebase offers a broad range of tools beyond analytics, such as push notifications, in-app messaging, and Crashlytics, making it a all-in-one tool for app development and management.
```
```suggestion
1. **It's a comprehensive toolkit:** Firebase offers a broad range of tools beyond analytics, such as push notifications, in-app messaging, and Crashlytics, making it an all-in-one tool for app development and management.
```

jtemperton commented on contents/blog/best-mobile-app-analytics-tools.mdx:
```diff
@@ -0,0 +1,327 @@
+
+2. **It's easy to use:** Reviewers appreciate how easy Firebase is to use, making app management simple.
+
+3. **It integrates with other Google services:** Firebase integrates smoothly with other Google services, providing a powerful development and analytics ecosystem.
```
we're kinda repeating ourselves here?

jtemperton commented on contents/blog/best-mobile-app-analytics-tools.mdx:
```diff
@@ -0,0 +1,327 @@
+
+### How much does Appsflyer cost?
+
+Since they focus on marketing analytics, Appsflyer charges $0.07 per conversion events. Examples of conversion are app installs, in-app purchases, and sign-ups.
```
```suggestion
Since it focuses on marketing analytics, Appsflyer charges $0.07 per conversion events. Examples of conversion are app installs, in-app purchases, and sign-ups.
```

jtemperton commented on contents/blog/best-mobile-app-analytics-tools.mdx:
```diff
@@ -0,0 +1,327 @@
+
+According to [G2 reviews](https://www.g2.com/products/appsflyer/reviews), companies use Appsflyer because:
+
+1. **They need comprehensive attribution insights:** Appsflyer enables marketers to understand which campaigns and channels are driving the best results
```
```suggestion
1. **They need comprehensive attribution insights:** Appsflyer enables marketers to understand which campaigns and channels are driving the best results.
```

jtemperton commented on contents/blog/best-mobile-app-analytics-tools.mdx:
```diff
@@ -0,0 +1,327 @@
+
+- 📊 **Product analytics:** Track user behavior, application performance, and key metrics with customizable dashboards
+
+- 🌐 **Privacy compliance:** Ensure compliance with GDPR, HIPAA, CCPA, and other privacy regulations,
```
```suggestion
- 🌐 **Privacy compliance:** Ensure compliance with GDPR, HIPAA, CCPA, and other privacy regulations.
```

jtemperton commented on contents/blog/best-mobile-app-analytics-tools.mdx:
```diff
@@ -0,0 +1,327 @@
+
+#### Key features
+
+- 📊 **Product analytics:** Track user behavior, application performance, and key metrics with customizable dashboards
```
```suggestion
- 📊 **Product analytics:** Track user behavior, application performance, and key metrics with customizable dashboards.
```

jtemperton commented on contents/blog/best-mobile-app-analytics-tools.mdx:
```diff
@@ -0,0 +1,327 @@
+
+- 💬 **Push notifications:** Engage users with personalized push notifications based on their behavior and preferences.
+
+- 🚨 **Crash analytics:** Monitor and resolve app crashes with detailed reports and real-time alerts
```
```suggestion
- 🚨 **Crash analytics:** Monitor and resolve app crashes with detailed reports and real-time alerts.
```

jtemperton commented on contents/blog/best-mobile-app-analytics-tools.mdx:
```diff
@@ -0,0 +1,327 @@
+
+### How much does Countly cost?
+
+Countly has different tiers based on number of active users. To give you an idea of how much they charge per active user, below is a list of their first four tiers:
```
```suggestion
Countly has different tiers based on number of active users. To give you an idea of how much it charges per active user, below is a list of its first four tiers:
```

jtemperton commented on contents/blog/best-mobile-app-analytics-tools.mdx:
```diff
@@ -0,0 +1,327 @@
+
+Based on [G2 reviews](https://www.g2.com/products/countly/reviews), companies choose Countly because:
+
+1. **It's focuses on privacy and compliance:** Countly ensures that companies can adhere to strict regulatory requirements.
```
```suggestion
1. **It's focussed on privacy and compliance:** Countly ensures that companies can adhere to strict regulatory requirements.
```

jtemperton reviewed (approved):
Squished a few typos, etc. As a more general point, and I'm not sure how much of an issue this is, I didn't get a strong sense throughout of this being a guide for the best _mobile app_ analytics tools. 

More callbacks to the specific functionality these products have for mobile app analytics would be really useful.

Lior539 commented:
> More callbacks to the specific functionality these products have for mobile app analytics would be really useful.

Yeah. Mobile analytics is not different from regular analytics, so none of the platforms have features specifically for mobile (except for Appsflyer, which helps track installs). Manoel mentioned to me that mixpanel and amplitude are the two most popular choices, and neither have mobile specific features

jtemperton commented:
> > More callbacks to the specific functionality these products have for mobile app analytics would be really useful.
> 
> Yeah. Mobile analytics is not different from regular analytics, so none of the platforms have features specifically for mobile (except for Appsflyer, which helps track installs). Manoel mentioned to me that mixpanel and amplitude are the two most popular choices, and neither have mobile specific features

this makes sense! in which case it's good to go imo

andyvan-ph commented on contents/blog/best-mobile-app-analytics-tools.mdx:
```diff
@@ -0,0 +1,327 @@
+
+import { ComparisonTable } from 'components/ComparisonTable'
+import { ComparisonRow } from 'components/ComparisonTable/row'
+
+When it comes to mobile apps, there are a bunch of different analytics tools to choose from, but each is built with a specific target audience in mind. In this guide, we’ll explore the options so that you can decide which is best for you.
```
```suggestion
```

andyvan-ph replied:
I think we can lose the intro on this one.

Lior539 commented on contents/blog/best-mobile-app-analytics-tools.mdx:
```diff
@@ -0,0 +1,325 @@
+---
+date: 2024-07-15
```
```suggestion
date: 2024-08-01
```
